spec/spec_helper_acceptance.rb

```diff
@@ -22,14 +22,20 @@
   c.before :suite do
     # Install module and dependencies
     hosts.each do |host|
-      if fact_on(host, 'osfamily') == 'RedHat'
+      if fact_on(host, 'os.family') == 'RedHat'
         # don't delete downloaded rpm for use with BEAKER_provision=no +
         # BEAKER_destroy=no
         on host, 'sed -i "s/keepcache=.*/keepcache=1/" /etc/yum.conf'
         # refresh check if cache needs refresh on next yum command
         on host, 'yum clean expire-cache'
+
+        major = fact_on(host, 'os.release.major')
+
+        if major == '8'
+          on host, puppet('resource', 'package', 'glibc-langpack-en', 'ensure=installed')
+        end
         # foreman nightly provides katello-cert-tools
-        host.install_package('https://yum.theforeman.org/releases/nightly/el7/x86_64/foreman-release.rpm')
+        host.install_package('https://yum.theforeman.org/releases/nightly/el#{major}/x86_64/foreman-release.rpm')
```

Comment:
Single quotes don't allow variable interpolation
```suggestion
        host.install_package("https://yum.theforeman.org/releases/nightly/el#{major}/x86_64/foreman-release.rpm")
```